### ISSUE TITLE
[Bug]: Problem with the recent update (v2.5.0)

### ISSUE THREAD

**Issue by Natlyz** [wardz/Diminish#26]
### What happened?

**Hello!** I'm getting these errors after the recent update:

> 1x Diminish\Diminish-v2.5.0.lua:203: attempt to index field 'DEFAULT_SETTINGS' (a nil value)
> [string "@Diminish\Diminish-v2.5.0.lua"]:203: in function 'InitDB'
> [string "@Diminish\Diminish-v2.5.0.lua"]:232: in function <Diminish\Diminish.lua:231>
> [string "=(tail call)"]: ?

> 1x Diminish\core\config.lua:48: bad argument #2 to 'format' (string expected, got nil)
> [string "=[C]:"]: in function 'format'
> [string "@Diminish\core\config.lua"]:48: in main chunk
> 
> Locals:
> (*temporary) = "Error: You're currently using the %s version of Diminish on a Retail client. You need to download the Retail version instead."
> (*temporary) = nil
> (*temporary) = "string expected, got nil"

I've updated with the Overwolf app, then manually from WoWInterface and Curse, by downlodaing the Diminish-v2.5.0.zip but that didn't solve the problem.

Thanks!

### Game Version

Retail

### Game Language

English

### Does the issue still occur when all other addons are temporarily disabled?

Yes, issue still occurs.

### Any additional info?

Disabling all other addons doesn't help.

**Comment by Road-block** [comment]
The actual first error for me is 
```
1x Diminish\core\config.lua:50: bad argument #2 to 'format' (string expected, got nil)
[string "=[C]"]: in function `format'
[string "@Diminish\core\config.lua"]:50: in main chunk

Locals:
(*temporary) = "Error: You're currently using the %s version of Diminish on a TBC client. You need to download the TBC version instead."
(*temporary) = nil
(*temporary) = "string expected, got nil"
```
then the DEFAULT_SETTINGS one posted at the top, then a bunch of null errors

Playing on a BCC enUS client.

**Comment by zaphon** [comment]
Same for me on Retail. I reverted back to prior version, issues gone.

**Comment by wardz** [comment]
Sorry been gone all day, will take a look now in a sec

**Comment by wardz** [comment]
Should be fixed now in v2.5.1 (tested classic+tbc, but not retail as i dont have it installed atm). Thanks for notifying.

**Comment by Natlyz** [comment]
The problem seems to be fixed for Retail. Thanks!
